torch/nn/intrinsic/qat/modules/conv_fused.py

```diff
@@ -79,27 +79,33 @@ def freeze_bn_stats(self):
         self.bn.training = False
         return self
 
+    # Note: this should be kept in sync with
+    # torch/csrc/jit/passes/quantization/qat_combine_conv_bn.cpp
     def _forward(self, input):
```

Comment:
the changes to this function do not change the logic, they just make it easier to inspect and reason about the resulting IR and write the equivalent IR in the JIT pass.